<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="plainText">
     <string>Model Definition</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>251</width>
      <height>192</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Nodes Amount</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Activation Function</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="add_layer_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Layer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_layer_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Layer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paint_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>paint something</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>261</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1127</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuproject">
    <property name="title">
     <string>project</string>
    </property>
    <addaction name="actionload_project"/>
    <addaction name="actionsave_project"/>
   </widget>
   <addaction name="menuproject"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionload_project">
   <property name="text">
    <string>load project</string>
   </property>
  </action>
  <action name="actionsave_project">
   <property name="text">
    <string>save project</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
